### PR TITLE
Note:

### PR DESCRIPTION
**Pull request by Taoli2018**
Fixed
just FYI
after check the code change history, this issue was not caused by recent changes,I rolled bacl all rcent changes , still can reproduce this issue,
 It happened long time ago, in case other clients has same issue, then  may refere to the fix here.

### CODE REVIEW

File: web/src/main/webapp/includes/global_functions_javascript.js [hunk 1]
```diff
@@ -1,3 +1,21 @@
+  function checkDataStatus() {
+
+        objImage=document.getElementById('status_top');
+        if (objImage != null && objImage.src.indexOf('images/icon_UnsavedData.gif')>0) {
+            return confirm('<fmt:message key="you_have_unsaved_data" bundle="${resword}"/>');
+        }
+
+        return true;
+    }
+    function gotoLink() {
```
Comment (svadla-oc) [inline web/src/main/webapp/includes/global_functions_javascript.js:10]: this function is not called from anywhere. Can you look into this?

**Review by svadla-oc — CHANGES_REQUESTED** [review]

**Comment by svadla-oc** [comment]
we need to create a PR for 3.17.x and assign fix version to the ticket. Also, please add the ticket number to the commit message

File: web/src/main/webapp/includes/global_functions_javascript.js (hunk 1 above)
Comment (Taoli2018) [reply web/src/main/webapp/includes/global_functions_javascript.js:10]: those 2 functions are called in many jsp pages, like initialDataEntryNw.jsp, and also can find same function definitions, but it's in html comments block

**Comment by Taoli2018** [comment]
please see new PR https://github.com/OpenClinica/OpenClinica/pull/3471

**Pull request closed without merging**